src/Console/Command/ProcessCommand.php

```diff
@@ -159,8 +159,9 @@ private function processFiles(array $fileInfos): void
             try {
                 $this->processFile($fileInfo, $i);
             } catch (Throwable $throwable) {
+                $this->consoleStyle->newLine();
                 throw new FileProcessingException(
-                    sprintf('Processing of "%s" file failed.', $fileInfo->getRealPath()),
+                    sprintf('Processing of "%s" file failed.', $fileInfo->getRelativePathname()),
```

Comment:
This path should be relative to `getcwd()`, so that inludes the `tests` directory and is eaier to find.

Reply:
I tried to simplify the pathname to do not include `tests`, as we passed to the `process` argument:
```bash
rector process tests/ --level phpunit-specific-method
```

Reply:
That is true, but user has to find file itselt and we don't want to force him to remember what was passed as argument and prepend it to each file.

It's common for CLI tools (PHPUnit, PHP CS Fixer, ECS, PHPStan) to report due to current working directory.